airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/SyncWorkflowImpl.java

```diff
@@ -58,7 +59,9 @@ public StandardSyncOutput run(final JobRunConfig jobRunConfig,
               .withCatalog(run.getOutputCatalog())
               .withResourceRequirements(syncInput.getDestinationResourceRequirements());
 
-          normalizationActivity.normalize(jobRunConfig, destinationLauncherConfig, normalizationInput);
+          final StandardNormalizationSummary normalizationSummary =
+              normalizationActivity.normalize(jobRunConfig, destinationLauncherConfig, normalizationInput);
+          persistActivity.persist(connectionId, run.withStandardNormalizationSummary(normalizationSummary));
```

Comment:
If you look at [these lines](https://github.com/airbytehq/airbyte/blob/ead33566776befc41d79b2f193895328e06f807b/airbyte-workers/src/main/java/io/airbyte/workers/WorkerApp.java#L198-L202) in the WorkerApp, you can see that the `persistActivity` here is initialized to the `PersistStateActivityImpl` class. Looking at [that class](https://github.com/airbytehq/airbyte/blob/ead33566776befc41d79b2f193895328e06f807b/airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/PersistStateActivityImpl.java#L29-L32), it just pulls the `state` field out of the StandardSyncOutput and persists that to the db. 

Since this doesn't depend on the `standardSyncSummary` or the `normalizationSummary` you are adding here, this call you added to `persistActivity.persist()` is not actually needed. This should be removed so that we aren't persisting state to the db twice.